### PR TITLE
Convert project to Maven and update dependencies

### PR DESCRIPTION
**Pull request by leocomelli**
Converted project to Maven. Use it like a maven dependency on your project:

``` XML
<dependency>
    <groupId>org.dbunit.dataset.builder</groupId>
    <artifactId>dbunit-datasetbuilder</artifactId>
    <version>1.0</version>
</dependency>
```

Updated dependencies
dbunit: 2.4.8 > 2.4.9
slf4j-*: 1.6.1 > 1.7.5
junit: 4.11

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.dbunit.dataset.builder</groupId>
+	<artifactId>dbunit-datasetbuilder</artifactId>
+	<version>1.0</version>
```
Comment (marcphilipp) [inline pom.xml]: This should be 1.0-SNAPSHOT, shouldn't it?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+	<name>DBUnint Data Set Builder</name>
+	<description>DBUnit - Dynamically Creating Data Sets Using Builders</description>
+
+	<licenses>
+		<license>
+			<name>GNU General Public License (GPL)</name>
```
Comment (marcphilipp) [inline pom.xml]: Originally, I had planned to integrate this code with the core DbUnit project. Thus, I decided for the same licence which at the time was LGPL. Has that changed?

**Comment by marcphilipp** [comment]
Looks very good, overall, thanks! Please give me a couple of days to review it thoroughly.

File: pom.xml (hunk 1 above)
Comment (leocomelli) [inline pom.xml]: Ok! I don't see any problem.

File: pom.xml (hunk 2 above)
Comment (leocomelli) [inline pom.xml]: Sorry! error caused by copy and paste.

**Pull request merged**